### ISSUE TITLE
can you share the training log?

### ISSUE THREAD

**Issue by Feynman1999** [yang-jin-hai/SAIN#4]
Hello, may I kindly ask if you can share the training logs? This would be helpful for me to replicate your work. Thank you!
Additionally, have you tried adjusting the weight ratios of different losses, and does it have a significant impact on the results?

**Comment by yang-jin-hai** [comment]
You can refer to #1 first.

**Comment by Feynman1999** [comment]
> You can refer to #1 first.

yes, i have refer it, at the same setting, I am currently training and I don't know the final result yet. 
But the phenomenon I have observed is that sometimes the loss suddenly increases by about twice, and then the validation set results decrease, but later converge again. Have you ever encountered this phenomenon?

**Comment by yang-jin-hai** [comment]
The validation results would generally increase steadily and may sometimes drop slightly. But the losses may be unstable.
